tests/test_engine.py

```diff
@@ -1125,33 +1093,6 @@ def flaky_function():
             StateType.COMPLETED,
         ]
 
-    async def test_task_timeouts_retry_properly(self, flow_run, orion_client):
```

Comment:
Frankly not sure what this test is doing but `test_task_timeout_deadline_is_reset_on_retry` should provide the same (and improved!) coverage